### PR TITLE
Fixed UI issues from Andres

### PR DESCRIPTION
**Pull request by danieltjason**
Fixed UI issues from Andres

### CODE REVIEW

File: src/scripts/script-update-block.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+          .where({
+            id: txs[i].id,
+          })
+          .execute();
+      }
+      if (txs[i].height && !txs[i].blockHash) {
+        const [hash] = await rpcClient.command([
+          {
+            method: 'getblockhash',
+            parameters: [102081],
```
Comment (crazyoptimist) [inline src/scripts/script-update-block.ts]: Pls fix hardcoded id

File: src/scripts/script-update-block.ts [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    .createQueryBuilder()
+    .select(['height', 'blockHash', 'id'])
+    .where(
+      `blockHash is null and (height is null or height < ${
+        Number(height) - 1
+      })`,
+    )
+    .getRawMany();
+  if (txs.length) {
+    for (let i = 0; i < txs.length; i += 1) {
```
Comment (crazyoptimist) [inline src/scripts/script-update-block.ts:31]: NB: `.forEach` or `for (const tx of txs)` are preferrable

File: src/scripts/script-update-block.ts [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            blockHash: hash,
+          })
+          .where({
+            id: txs[i].id,
+          })
+          .execute();
+      }
+    }
+  }
+  console.log('end update block>>>>', new Date().getMilliseconds(), 's');
```
Comment (crazyoptimist) [inline src/scripts/script-update-block.ts]: Inconsistent logs: before it was `getSeconds`, here it is `getMilliseconds`. Why do you need to log milliseconds?

File: src/scripts/script-update-block.ts [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+}
+
+createConnection().then(updateUnconfirmedBlocks);
```
Comment (crazyoptimist) [inline src/scripts/script-update-block.ts:82]: Script won't exit because of opened connection, don't we need to close connection in the end?

**Review by crazyoptimist — CHANGES_REQUESTED** [review]

File: src/scripts/script-update-block.ts (hunk 2 above)
Comment (danieltjason) [reply src/scripts/script-update-block.ts:31]: I think, it's better in terms of performance
https://medium.com/tech-tajawal/loops-performances-in-node-js-9fbccf2d6aa6

File: src/scripts/script-update-block.ts (hunk 1 above)
Comment (danieltjason) [reply src/scripts/script-update-block.ts]: I just fixed it.

https://github.com/pastelnetwork/pastel-explorer-backend/pull/40/commits/84218141b698511a3f37b59dde851db1a7aa902e

File: src/scripts/script-update-block.ts (hunk 3 above)
Comment (danieltjason) [reply src/scripts/script-update-block.ts]: I just fixed it.
https://github.com/pastelnetwork/pastel-explorer-backend/pull/40/commits/84218141b698511a3f37b59dde851db1a7aa902e

File: src/scripts/script-update-block.ts (hunk 4 above)
Comment (danieltjason) [reply src/scripts/script-update-block.ts:82]: I just fixed it
https://github.com/pastelnetwork/pastel-explorer-backend/pull/40/commits/84218141b698511a3f37b59dde851db1a7aa902e

File: src/scripts/script-update-block.ts (hunk 2 above)
Comment (crazyoptimist) [reply src/scripts/script-update-block.ts:31]: Agree, makes sense with large amount of data

**Review by crazyoptimist — APPROVED** [review]

**Review by Dicklesworthstone — APPROVED** [review]

**Pull request merged**